src/Adapter/Module/ModuleZipManager.php

```diff
@@ -164,8 +164,8 @@ public function storeInModulesFolder($source)
         );
         $this->eventDispatcher
             ->dispatch(
-                ModuleZipManagementEvent::DOWNLOAD,
-                new ModuleZipManagementEvent($this->getSource($source))
+                new ModuleZipManagementEvent($this->getSource($source)),
```

Comment:
That's weird did the `dispatch` change? Or was it incorrectly used from the start?

Reply:
`dispatch` changed 😅 The event name is now optional!

Reply:
So optional that it's now in second position 😅
So ideally the event name is store by the event itself I guess?

Reply:
Not exactly, you can use the class type of the event directly: [https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching](https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching)